src/main/java/org/broadinstitute/dsm/route/PatchRoute.java

```diff
@@ -42,6 +42,7 @@
 
```

Comment:
@pegahtah @SimoneMaiwald guys please pay attention to this comment: https://broadinstitute.slack.com/archives/C02BAHMHE1G/p1639065811003800?thread_ts=1639065448.003300&cid=C02BAHMHE1G